Branch-SDK/Branch-SDK/BranchUniversalObject.h

```diff
@@ -42,7 +43,7 @@ typedef NS_ENUM(NSInteger, ContentIndexMode) {
 - (void)showShareSheetWithShareText:(NSString *)shareText andCallback:(callback)callback;
 - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties andShareText:(NSString *)shareText fromViewController:(UIViewController *)viewController andCallback:(callback)callback;
 - (void)listOnSpotlight;
-- (void)listOnSpotlightWithCallback:(callbackWithUrl)callback;
+- (void)listOnSpotlightWithCallback:(callbackWithUrlAndSpotlightIdentifier)callback;
```

Comment:
@danwalkowski this is a breaking change; this is what I'm referring to. Although you only added a new method on `BNCContentDiscoveryManager`, you replaced this method on `BranchUniversalObject`. If we want backwards compatibility, you should leave the old method `- (void)listOnSpotlightWithCallback:(callbackWithUrl)callback;`. Then the internals of `BNCContentDiscoveryManager` should allow for either callback format